aikau/src/main/resources/alfresco/search/FacetFilter.js

```diff
@@ -253,7 +253,8 @@ define(["dojo/_base/declare",
        */
       _updateHash: function alfresco_search_FacetFilter___updateHash(fullFilter, mode) {
          // Get the existing hash and extract the individual facetFilters into an array
-         var aHash = hashUtils.getHash(),
+         var aHashString = hashUtils.getHashString(),
+             aHash = ioQuery.queryToObject(aHashString),
```

Comment:
As above (and already duplication of the functionality, which is a reason to modify it in the utility)